packages/foam-vscode/src/services/config.ts

```diff
@@ -0,0 +1,17 @@
+import { workspace } from "vscode";
+import { FoamConfig, createConfigFromFolders } from "foam-core";
+import { getIgnoredFilesSetting } from "../settings";
+
+// TODO this is still to be improved - foam config should
+// not be dependent on vscode but at the moment it's convenient
+// to leverage it
+export const getConfigFromVscode = (): FoamConfig => {
```

Comment:
This goes back to the discussion of how coupled Foam and VSC should be. Maybe this could be escalated as an issue striving for clearer goals?

Reply:
Sounds good to me, can you please create one and start the conversation there?